ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineForkchoiceUpdatedTest.java

```diff
@@ -244,7 +247,8 @@ public void shouldReturnValidWithoutFinalizedWithPayload() {
     BlockHeader mockHeader = blockHeaderBuilder.buildHeader();
     when(mergeCoordinator.getOrSyncHeadByHash(mockHeader.getHash(), Hash.ZERO))
         .thenReturn(Optional.of(mockHeader));
-    when(mergeCoordinator.latestValidAncestorDescendsFromTerminal(mockHeader)).thenReturn(true);
+    if (validateTerminalPoWBlock())
```

Comment:
nit: think it's clearer (and more conventional with besu's code style) to use braces after the if () {...}

Reply:
Updated